### PR TITLE
[Bad Guys] Reorder projects page

### PR DESCRIPTION
**Pull request by mikeharv**
Builds on the work started in:
- https://github.com/code-dot-org/code-dot-org/pull/45496

We need to enable starting new "The Bad Guys" projects on the Projects Page and reorder existing projects. @KylieModen and I also think it makes sense to update the default set of projects for under 13 users *not in sections*.

The new categories presented here were shared at the 3/31/21 T+L+P+E sync and approved by those in attendance.

Proposed project categories for users who are **13 or over** or **in a section**:
160919723-ae2602b1-fff0-4fc1-b1ae-b71d7f2a6031161104094-a44d72f2-6559-4434-8123-6c44a975ed4f

Proposed project categories for users who are **under 13** and **not in a section**:
160919773-95a305aa-0bc7-4612-83c5-b4c0f9fadf73

**Note:** *Today, thedefault list for these users includes Poetry and Minecraft Designer instead of Play Lab.*
160919849-fa264c47-eb26-4e6b-aa2d-34ac58cf9de4

## Links

jira tickets: 
- [STAR-2192:[Bad Guys] Enable The Bad Guys Project in /Projects](https://codedotorg.atlassian.net/browse/STAR-2192)
- [STAR-2193: Reorder projects page](https://codedotorg.atlassian.net/browse/STAR-2193)

### CODE REVIEW

**Comment by epeach** [comment]
(Is this ready for review or still a draft?)

**Comment by mikeharv** [comment]
> (Is this ready for review or still a draft?)

This is still technically a draft because I'm waiting for T&L approval for the new categories. I'll ping the team when ready, and perhaps not add reviewers until then next time.

**Review by KylieModen — APPROVED** [review]
LGTM - order reviewed in T&L meeting

**Comment by epeach** [comment]
Can you add a note that points of the differences between the under13/over13 images? It's a bit hard to see the differences. Thanks!

File: apps/src/templates/projects/StartNewProject.jsx
Comment (epeach) [inline apps/src/templates/projects/StartNewProject.jsx:49]: Moving these lists to global variables is a really good clean-up!

**Review by epeach — APPROVED** [review]
:+1:

**Comment by mikeharv** [comment]
> Can you add a note that points of the differences between the under13/over13 images? It's a bit hard to see the differences. Thanks!

@epeach Added annotations to images to highlight differences!

**Comment by epeach** [comment]
thanks

File: apps/src/templates/projects/StartNewProject.jsx
Comment (breville) [inline apps/src/templates/projects/StartNewProject.jsx:51]: Just an idea: should we prefix/suffix all of these with something like `_PROJECT_TYPES`?

**Pull request merged**
